packages/conventional-changelog-writer/README.md

```diff
@@ -85,7 +85,7 @@ Type: `string`
 
 Type: `boolean` Default: `semver.patch(context.version) !== 0`
 
-By default, this value is true if `version`'s patch is `0`.
+By default, this value is true if `version`'s patch is not `0`.
```

Comment:
Unless I was reading this incorrectly, seemed like this was a typo. `isPatch` should be false if the patch is 0, not the other way around